.circleci/config.yml

```diff
@@ -223,6 +226,12 @@ jobs:
                       cd python-packages
                       ./parallel_without_sra_client coverage run setup.py test
                       ./build_docs
+            - run:
+                  command: |
+                      # copy generated wrappers into contract_wrappers/build,
+                      # JUST so CircleCI will persist them as build artifacts.
+                      cd python-packages/contract_wrappers/src/zero_ex
+                      for i in contract_wrappers/[^__]*/; do mkdir -p ../../build/$i; cp $i/__init__.py ../../build/$i; done
```

Comment:
I'd prefer not to, because it would create an unnecessary level of indirection/abstraction.  Better to keep everything inline here so that the reader of this file doesn't have to jump somewhere else in order to follow what's happening.  If we were re-using that logic somewhere else (for example from a `setup.py` or something) then I would be for this, but as of right now this is the only place this logic needs to live.